packages/tsconfig-rbflabs/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ts-config-rbflabs",
+  "name": "tsconfig-rbflabs",
```

Comment:
I don't see much value in sharing tsconfigs like npm package, do you? I think it is used here mainly because the turbo monorepo uses it like this.

Eslint package name is important, but the organization prefix is possible. I will add that.

Reply:
Yeah, it only makes sense to publish it once we want to use it outside of this monorepo. I thought that when you talked about publishing packages you meant these config packages that hadn't been published so far.